integration-tests/__tests__/failures.test.js

```diff
@@ -18,30 +18,6 @@ const normalizeDots = text => text.replace(/\.{1,}$/gm, '.');
 
 SkipOnWindows.suite();
 
-const cleanupStackTrace = stderr => {
```

Comment:
Unexpected that I was able to remove this 😛

Reply:
/cc @aaronabramov

Reply:
oh.. i completely forgot about it 😀
i'm glad i didn't write a `TODO: remove after it lands` comment that dated early 2016 or something haha